boards/PDM/Inc/Io/configs/Io_EfuseConfig.h

```diff
@@ -0,0 +1,463 @@
+#define PWM_CLOCK_SOURCE_EXTERNAL (0x00 << CLOCK_INT_S_SHIFT)
+
+#define PWM_EXTERNAL_CLOCK_DIVIDER_256 (0x00 << PR_S_SHIFT)
+#define PWM_EXTERNAL_CLOCK_DIVIDER_512 (0x01 << PR_S_SHIFT)
+
+// Feature definitions in RETRY_s register
+#define AUTO_RETRY_PERIOD_T_AUTO_00 \
+    ((0x00 << AUTO_PERIOD_1_S_SHIFT) | (0x00 << AUTO_PERIOD_0_S_SHIFT))
+#define AUTO_RETRY_PERIOD_tAUTO_01 \
+    ((0x00 << AUTO_PERIOD_1_S_SHIFT) | (0x01 << AUTO_PERIOD_0_S_SHIFT))
+#define AUTO_RETRY_PERIOD_T_AUTO_10 \
+    ((0x01 << AUTO_PERIOD_1_S_SHIFT) | (0x00 << AUTO_PERIOD_0_S_SHIFT))
+#define AUTO_RETRY_PERIOD_T_AUTO_11 \
+    ((0x01 << AUTO_PERIOD_1_S_SHIFT) | (0x01 << AUTO_PERIOD_0_S_SHIFT))
+
+#define AUTO_RETRY_NUMBER_16 (0x00 << RETRY_UNLIMITED_S_SHIFT)
+#define AUTO_RETRY_NUMBER_INFINITE (0x01 << RETRY_UNLIMITED_S_SHIFT)
+#define AUTO_RETRY_BULB_ENABLED (0x00 << RETRY_S_SHIFT)
+#define AUTO_RETRY_BULB_DISABLED (0x01 << RETRY_S_SHIFT)
+#define AUTO_RETRY_DC_MOTOR_ENABLED (0x01 << RETRY_S_SHIFT)
+#define AUTO_RETRY_DC_MOTOR_DISABLED (0x00 << RETRY_S_SHIFT)
+
+#define RANDOM_CURRENT_OFFSET_ADD (0x01 << OFP_S_SHIFT)
+#define RANDOM_CURRENT_OFFSET_SUBTRACT (0x00 << OFP_S_SHIFT)
+
+// Feature definitions in GCR register
+#define OVERVOLTAGE_PROTECTION_ENABLE (0x00 << OV_DIS_SHIFT)
+#define OVERVOLTAGE_PROTECTION_DISABLE (0x01 << OV_DIS_SHIFT)
+
+#define CSNS_FUNCTION_DISABLED \
+    ((0x00 << CSNS1_EN_SHIFT) | (0x00 << CSNS0_EN_SHIFT))
+#define CSNS_FUNCTION_CURRENT_CH0 \
+    ((0x00 << CSNS1_EN_SHIFT) | (0x01 << CSNS0_EN_SHIFT))
+#define CSNS_FUNCTION_CURRENT_CH1 \
+    ((0x01 << CSNS1_EN_SHIFT) | (0x00 << CSNS0_EN_SHIFT))
+#define CSNS_FUNCTION_TEMPERATURE \
+    ((0x01 << CSNS1_EN_SHIFT) | (0x01 << CSNS0_EN_SHIFT))
+#define CSNS_FUNCTION_CURRENT_SUM \
+    ((0x00 << CSNS1_EN_SHIFT) | (0x01 << CSNS0_EN_SHIFT))
+
+#define VDD_FAILURE_DETECTION_ENABLE (0x01 << VDD_FAIL_EN_SHIFT)
+#define VDD_FAILURE_DETECTION_DISABLE (0x00 << VDD_FAIL_EN_SHIFT)
+
+#define WATCHDOG_DISABLE (0x01 << WD_DIS_SHIFT)
+#define WATCHDOG_ENABLE (0x00 << WD_DIS_SHIFT)
+
+#define TRACK_AND_HOLD_CURRENT_ENABLE (0x01 << T_H_EN_SHIFT)
+#define TRACK_AND_HOLD_CURRENT_DISABLE (0x00 << T_H_EN_SHIFT)
+
+#define PARALLEL_MODE_ENABLE (0x01 << PARALLEL_SHIFT)
+#define PARALLEL_MODE_DISABLE (0x00 << PARALLEL_SHIFT)
+
+#define PWM_CHANNEL_0_ENABLE (0x01 << PWM_EN_0_SHIFT)
+#define PWM_CHANNEL_0_DISABLE (0x00 << PWM_EN_0_SHIFT)
+
+#define PWM_CHANNEL_1_ENABLE (0x01 << PWM_EN_1_SHIFT)
+#define PWM_CHANNEL_1_DISABLE (0x00 << PWM_EN_1_SHIFT)
+
+// Configuration values used for initializing the registers
+
+// Global Configuration
+#define GCR_CONFIG                                                           \
+    (PWM_CHANNEL_0_DISABLE | PWM_CHANNEL_1_DISABLE | PARALLEL_MODE_DISABLE | \
+     TRACK_AND_HOLD_CURRENT_DISABLE | WATCHDOG_DISABLE |                     \
+     VDD_FAILURE_DETECTION_ENABLE | CSNS_FUNCTION_DISABLED |                 \
+     OVERVOLTAGE_PROTECTION_ENABLE)
+
+// Channel  Configurations
```

Comment:
```suggestion
// Channel Configurations
```

Reply:
Actually how are these channel specific?

Reply:
They could be different, right now for simplicity the same config is applied to both channels